tests/foreman/cli/test_remoteexecution.py

```diff
@@ -105,8 +104,8 @@ def test_positive_run_default_job_template_by_ip(self, fixture_vmsetup):
     @pytest.mark.tier3
     @pytest.mark.pit_client
     @pytest.mark.pit_server
-    @pytest.mark.parametrize('fixture_vmsetup', [{'nick': 'rhel7'}], ids=['rhel7'], indirect=True)
-    def test_positive_run_job_effective_user_by_ip(self, fixture_vmsetup):
+    @pytest.mark.rhel_ver_list([7, 8, 9])
```

Comment:
No rhel 9 support, though without it in `supportability.yaml` having it here shouldn't actually do anything.